### DISCUSSION TITLE
Publically available data from api end point from pocketbase?

### DISCUSSION THREAD

**Discussion by taimoor-shan** [pocketbase/pocketbase#1699] [Q&A]
Can we make our endpoints publically available i,e without headers?
Like blog

**Comment by ganigeorgiev** [comment]
You can change the collection access rules from the
"Admin UI > {your collection} > Edit collection (settings cogwheel) > API rules". Leaving a rule empty grant access to everyone to perform the related action.

More information about the collections you can find in https://pocketbase.io/docs/collections/.

More information about the API rules you can find in https://pocketbase.io/docs/api-rules-and-filters/.
